README.md

```diff
@@ -13,29 +13,29 @@
 #### Supported options:
 
 * `--help, -help, help, -h`
- * Print help and exit.
- * Required: no.
+  * Print help and exit.
+  * Required: no.
 * `--emulator-name`
- * Name for the emulator, i.e. `test_emulator_1`.
- * Required: **yes**.
+  * Name for the emulator, i.e. `test_emulator_1`.
+  * Required: **yes**.
```

Comment:
With this options count you can split them to two subsections — `Required` and `Optional`. This way this will be a one-level list.